src/storage/interfaces/IBlockStorageModule.ts

```diff
@@ -0,0 +1,14 @@
+import {
+    BlockConfirmationStruct,
+    SignedBlockStruct
+} from "@typechain-types/contracts/V1/DataTypes";
+import { BlockHash } from "@/types/storage";
+
+export interface IBlockStorageModule {
+    getPreviousBlockHash(
+        forkCnt: number,
+        transactionCnt: number
+    ): BlockHash | undefined;
+    getLatestBlock(): SignedBlockStruct;
+    getLatestBlockConfirmation(): BlockConfirmationStruct;
+}
```

Comment:
In general not sure why all these specific interfaces exist 'IBlock...', 'IExit...' - the point of the interface is to be general - if iStorage is something that all storage components should implement than yes, otherwise no. Then it's just better to have the specific storage component (e.g. BlockStorage) define its interface and implementation

Reply:
yeah it first i had them inside their respective storage files. I figured i would extract them in a seperate folder just to be cleaner (habit of C#)

Reply:
Nah i agree its overengineering, no real reason to have them i wil lremove it